src/helpers/errorsHandling/CustumError.ts

```diff
@@ -0,0 +1,12 @@
+class CustumError extends Error {
+  status:number
+  message:string
+
+  constructor (status:number, message:string) {
+    super(message)
+    this.status = status
+    this.message = message
```

Comment:
i think this line is not necessary because line 6 binds the message

Reply:
do you mean line 8 ? @aaamra

Reply:
yes @Saeed99Madi